modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -11,52 +20,116 @@ If you do not need to review the JSON files that the `ccoctl` tool creates befor
 Otherwise, you can create the AWS resources individually.
 
 //to-do if possible: xref to modules/cco-ccoctl-creating-individually.adoc for `create the AWS resources individually`
+endif::aws[]
+ifdef::alibabacloud[]
+[id="cco-ccoctl-creating-at-once_{context}"]
+= Creating credentials for {product-title} components with the ccoctl tool
+
+You can use the `ccoctl alibabacloud create-ram-users` command to automate the creation of Alibaba Cloud RAM users and policies for each in-cluster component.
+endif::alibabacloud[]
 
 [NOTE]
 ====
-By default, `ccoctl` creates objects in the directory in which the commands are run. To specify a directory, use the `--output-dir` flag. This procedure uses `_<path_to_ccoctl_output_dir>_` to refer to this location.
+By default, `ccoctl` creates objects in the directory in which the commands are run. To specify a directory, use the `--output-dir` flag. This procedure uses `<path_to_ccoctl_output_dir>` to refer to this location.
 ====
 
 .Prerequisites
 
 * Extract and prepare the `ccoctl` binary.
+ifdef::alibabacloud[]
+* Obtain the AccessKey (AK) of the RAM user you wish to use.
+* Add the AcessKey ID (`access_key_id`) and AccessKey secret (`access_key_secret`) into the link:https://github.com/aliyun/alibaba-cloud-sdk-go/blob/master/docs/2-Client-EN.md#2-credentials-file[`~/.alibabacloud/credentials` file] on your local computer. The Cloud Credential Operator (CCO) utility uses these credentials when processing `Credential Request` objects. 
```

Comment:
The [best match when searching for alternatives](https://www.alibabacloud.com/search?type=doc&q=alibabacloud%2Fcredentials) I found for a reference pointing to official documentation was here:
[2. Use the configuration file](https://www.alibabacloud.com/help/en/doc-detail/311667.htm#h2-sls-mfm-3p3)